code/script.js

```diff
@@ -222,104 +227,189 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// Function: setSecret randomly selects a person from the charachter array (this is the secret person).
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log('secret person;', secret) //This console.log allows me to see who the secret person is for each game. 
 }
 
-// This function to start (and restart) the game
+// Function: starts the game. 
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  charactersInPlay = CHARACTERS // Here we're setting charactersInPlay array to be all the characters to start with
```

Comment:
The start function could be extended by including the `generateBoard()` and `setSecret()` functions inside it. This way all you would need to run your program is call `start()` on row 391. 
It may also be the reason the event listeners on `playAgainBtn` and `restartButton` don't work (if you tried to call start upon click) - since the only thing that happens now with `start()` is that the character array is assigned to charactersInPlay.